### PR TITLE
fix(linux): only pass tag if we just created one

### PR DESCRIPTION
**Pull request by ermshiperete**
Part of #4160

### CODE REVIEW

File: resources/build/trigger-builds.inc.sh [hunk 1]
```diff
@@ -66,6 +66,12 @@ function triggerJenkinsBuild() {
     FORCE=", \"force\": true"
   fi
 
+  local TAG=""
+  # This will only be true if we created and pushed a tag
+  if [ "$action" == "commit" ]; then
```
Comment (mcdurdin) [inline resources/build/trigger-builds.inc.sh:71]: This appears to have broken the build: https://build.palaso.org/viewLog.html?buildId=245552&buildTypeId=Keyman_Test&tab=buildLog&_focus=138

Comment (darcywong00) [reply resources/build/trigger-builds.inc.sh:71]: `$action` looks to be unbound

Comment (ermshiperete) [reply resources/build/trigger-builds.inc.sh]: fixed

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**